programs/survival/horse_stats.sc

```diff
@@ -0,0 +1,24 @@
+// Print the stats of the horse the player is looking at
+
+__config() -> ({ 'stay_loaded' -> true });
+
+__command() -> (
+	target = query(player(), 'trace', 4.5, 'entities');
+	if (target == 'Horse' || target == 'Donkey' || target == 'Mule',
```

Comment:
use target~'type' since named mobs will display differently here

Reply:
Thanks! I overlooked query(e, 'type') in the API, though I did find the uppercase suspicious.